src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/BailNotificationGeneratorConfiguration.java

```diff
@@ -30,4 +32,21 @@ public List<BailNotificationGenerator> submitApplicationHearingCentreNotificatio
             )
         );
     }
+
+    @Bean("uploadSummaryNotificationGenerator")
+    public List<BailNotificationGenerator> uploadSummaryNotificationGenerator(
+            AdminOfficerBailSummaryUploadedPersonalisation adminOfficerBailSummaryUploadedPersonalisation,
+            LegalRepresentativeBailSummaryUploadedPersonalisation legalRepresentativeBailSummaryUploadedPersonalisation,
+            NotificationSender notificationSender,
+            BailNotificationIdAppender notificationIdAppender) {
+
+        return Arrays.asList(
+                new BailEmailNotificationGenerator(
+                        newArrayList(adminOfficerBailSummaryUploadedPersonalisation,
+                                legalRepresentativeBailSummaryUploadedPersonalisation),
```

Comment:
Did you test the scenario when there in No LR, does this Personalisation get triggered?